workload_mapper/mariadb/container/scripts/config_mariadb.sh

```diff
@@ -46,4 +46,6 @@ if [ -z "$DB_FILES" ]; then
 fi
 
 # Don't run this again.
-rm -f /scripts/config_mariadb.sh
+# rm -f /scripts/config_mariadb.sh
```

Comment:
Is this commented out intentionally? If yes, I think it should rather be removed.